.bazelrc

```diff
@@ -0,0 +1,14 @@
+# Include system libraries
```

Comment:
I was expecting to see a Makefile from which you source these options, and the Makefile being deleted in this PR. I don't find any Makefile in the repo though ?

Reply:
OK. I was hoping to see that https://github.com/opencomputeproject/ocp-diag-sat/pull/3/files. Would help to understand these changes.